src/zope/component/tests/test_hookable.py

```diff
@@ -68,62 +68,38 @@ def bar():
     def test_cant_assign_original(self):
         from zope.component.hookable import hookable
         def foo():
-            pass
+            raise NotImplementedError()
```

Comment:
Wouldn't we be better off just adding `pass` to the list of lines excluded by coverage.